web-ng/src/app/components/project-header/project-header.component.html

```diff
@@ -30,12 +30,18 @@
         <mat-icon>more_vert</mat-icon>
       </button>
       <img
+        *ngIf="user.photoURL; else defaultPhoto"
         src="{{ user.photoURL }}=s64"
-        alt="{{ user.displayName }}"
         class="user-thumb"
-        style="cursor: pointer;"
         (click)="openProfileDialog($event)"
       />
+      <ng-template #defaultPhoto>
+        <span
+          [attr.data-letters]="user.displayName.charAt(0)"
```

Comment:
How does this actually work? What component causes the letter to be read from the data-letters and attribute? And why do we need :before pseudo-elements in CSS?

Reply:
Remove pseudo attr.